packages/aws-lambda/src/wrapper.js

```diff
@@ -93,8 +95,13 @@ function shimmedHandler(originalHandler, originalThis, originalArgs, _config) {
         alias,
         runtime: 'nodejs',
         functionName: context.functionName,
-        functionVersion: context.functionVersion
+        functionVersion: context.functionVersion,
+        reqId: context.awsRequestId
```

Comment:
This is unrelated to this PR but I figured that having the request ID in the span would be actually quite helpful for correlating a trace in Instana with cloudwatch logs, so I added it.